### PR TITLE
Remove pre/postinstall scripts

### PR DESCRIPTION
**Pull request by webpro**
99.9% we run `pnpm install` we don't need any of this :)

### CODE REVIEW

File: packages/libs/chainwebjs/package.json [hunk 1]
```diff
@@ -52,7 +52,6 @@
     "format": "pnpm run /^format:.*/",
     "format:md": "remark README.md -o --use @kadena-dev/markdown",
     "format:src": "prettier config src --write",
-    "preinstall": "npx only-allow pnpm",
```
Comment (webpro) [inline packages/libs/chainwebjs/package.json:55]: We define this in the root `package.json` already.

File: packages/libs/chainweb-node-client/package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "format:md": "remark README.md -o --use @kadena-dev/markdown",
     "format:src": "prettier config openapi src --write",
     "generate:openapi-types": "echo 'openapi specs needs fixes' # openapi-typescript \"./openapi/*.json\" --output ./src/openapi",
-    "postinstall": "pnpm run generate:openapi-types",
```
Comment (alber70g) [inline packages/libs/chainweb-node-client/package.json:38]: Shouldn't this be part of the build script too?

```
"build": "pnpm run generate:openapi-types && heft build --clean",
```

File: packages/libs/chainwebjs/package.json (hunk 1 above)
Comment (alber70g) [reply packages/libs/chainwebjs/package.json:55]: Does this have an effect on clients installing chainwebjs?

**Review by alber70g — APPROVED** [review]

File: packages/libs/chainwebjs/package.json (hunk 1 above)
Comment (webpro) [reply packages/libs/chainwebjs/package.json:55]: Yes, they are no longer forced to use pnpm

File: packages/libs/chainweb-node-client/package.json (hunk 2 above)
Comment (webpro) [reply packages/libs/chainweb-node-client/package.json:38]: Didn't bother too much, this isn't doing anything anyway (just `echo`)

**Pull request merged**